src/ape_test/accounts.py

```diff
@@ -107,7 +102,7 @@ def reset(self):
 class TestAccount(TestAccountAPI):
     index: int
     address_str: str
-    private_key: str
+    signer: ApeSigner
```

Comment:
Hmm, what would be wrong with subclassing `ApeSigner` here?

Reply:
nothing, just had to also use the `TestAccountAPI` mixin, which should be fine